test/core/security/grpc_tls_crl_provider_test.cc

```diff
@@ -174,67 +201,69 @@ TEST_F(DirectoryReloaderCrlProviderTest,
        DirectoryReloaderCrlLookupMissingIssuer) {
   auto provider =
       CreateCrlProvider(kCrlDirectory, std::chrono::seconds(60), nullptr);
-  ASSERT_TRUE(provider.ok());
-
+  ASSERT_TRUE(provider.ok()) << provider.status();
   CertificateInfoImpl bad_cert("BAD CERT");
   auto crl = (*provider)->GetCrl(bad_cert);
   ASSERT_EQ(crl, nullptr);
 }
 
 TEST_F(DirectoryReloaderCrlProviderTest, DirectoryReloaderReloadsAndDeletes) {
-  int refresh_duration = 60;
-  auto provider =
-      CreateCrlProvider(std::chrono::seconds(refresh_duration), nullptr);
-  ASSERT_TRUE(provider.ok());
+  const std::chrono::seconds kRefreshDuration(60);
+  absl::StatusOr<std::vector<std::string>> files = std::vector<std::string>();
```

Comment:
I don't think this needs to be a `StatusOr<>`.  It can just be a vector, and in the case where you want to pass a non-OK status, you can just pass the status directly to `SetFilesInDirectory()` instead of using this temporary variable.

Same in all tests.

Reply:
Fixed throughout